Interrupts/linux-interrupts-2.md

```diff
@@ -78,12 +78,12 @@ As you can remember the 32-bit entry point is in the [arch/x86/boot/compressed/h
 But the 32-bit mode entry point is the second file in our case. The first file is not even compiled for `x86_64`. Let's look at the [arch/x86/boot/compressed/Makefile](https://github.com/torvalds/linux/blob/16f73eb02d7e1765ccab3d2018e0bd98eb93d973/arch/x86/boot/compressed/Makefile):
 
 ```
-vmlinux-objs-y := $(obj)/vmlinux.lds $(obj)/head_$(BITS).o $(obj)/misc.o \
+vmlinux-objs-y := $(obj)/vmlinux.lds $(obj)/kernel_info.o $(obj)/head_$(BITS).o \
```

Comment:
why this was changed? https://github.com/torvalds/linux/blob/16f73eb02d7e1765ccab3d2018e0bd98eb93d973/arch/x86/boot/compressed/Makefile#L72 does not contain `kernel_info.o` object file

Reply:
Hmm, that is strange, on bootlin the kernel_info.o object can be found : https://elixir.bootlin.com/linux/latest/source/arch/x86/boot/compressed/Makefile#L76 . I also found that change on my local linux next version.
Thanks for the review :+1: 
I drop that change.

Reply:
Ah wait a second: https://github.com/torvalds/linux/blob/master/arch/x86/boot/compressed/Makefile here kernel_info can also be found.